src/pysoem/pysoem.pyx

```diff
@@ -53,6 +53,64 @@ ECT_COEDET_PDOCONFIG = 0x08
 ECT_COEDET_UPLOAD    = 0x10
 ECT_COEDET_SDOCA     = 0x20
 
+cdef class _Timeouts:
```

Comment:
To align with the other class names I would prefer to call this CdefTimeouts. Because the class is not exposed via the `__init__.py` it will not be visible anyway, so no underscore needed.